data/crac-creation/crac-creator-csa-profiles/src/test/java/com/farao_community/farao/data/crac_creation/creator/csa_profile/crac_creator/remedial_action/InjectionSetPointActionCreationTest.java

```diff
@@ -89,7 +89,7 @@ void testImportInjectionSetPointActions() {
         // on-state-included-curative-remedial-action (on state)
         NetworkAction ra3 = cracCreationContext.getCrac().getNetworkAction("on-state-included-curative-remedial-action");
         assertEquals("RA3", ra3.getName());
-        assertEquals(UsageMethod.FORCED, ra3.getUsageRules().iterator().next().getUsageMethod());
+        assertEquals(UsageMethod.AVAILABLE, ra3.getUsageRules().iterator().next().getUsageMethod());
```

Comment:
for now this is supposed to stay FORCED
@bqth29 can confirm

Reply:
Indeed, but we thought TSOs probably don't understand what `FORCED` means and will change their specifications. 

We were supposed to have TSOs response on this subject last week but we didn't :confused:

Moreover, with this dev we ignore `FORCED` remedial actions other than for `Instant.AUTO` so it won't work :disappointed: